### PR TITLE
Update TFJob docs with information about monitoring.

### PR DESCRIPTION
**Pull request by jlewi**
* Move over the documentation in kubeflow/tf-operator/README.md and
  update it for v1alpha2.

* Add a what is TFJob section that describes the TFJob
  * Provide information about the spec
  * Provide information about exit codes.

* Provide stackdriver sections for both the legacy and new Kubernetes
  stackdriver.

* Add a troubleshooting section.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/107)

### CODE REVIEW

File: content/docs/guides/components/tftraining.md [hunk 1]
```diff
@@ -74,14 +204,14 @@ Typically you will want to change the following values
 
 1. Attach PV's if you want to use PVs for storage.
 
-## Submitting a TensorFlow training job
+### Submitting a TensorFlow training job
```
Comment (abhi-g) [inline content/docs/guides/components/tftraining.md]: Lines 207 - 271 seem to be duplicate of lines 143-206.

**Comment by kunmingg** [comment]
/lgtm
Thanks, very detailed docs.

**Pull request merged**